fan_control.py

```diff
@@ -226,7 +227,21 @@ def main():
 
         time.sleep(config['general']['interval'])
 
+
+def graceful_shutdown(signalnum, frame):
+    strsignal = signal.strsignal(signalnum)
+    print(f"setting hosts back to automatic fan control because we received the signal: {strsignal}")
+
+    for host in config['hosts']:
+        set_fan_control("automatic", host)
+
+    sys.exit(128 + signalnum)
```

Comment:
Exiting with code `0` may play nicer with init systems, this is a clean termination after all.

Reply:
Yeah that's true, I'll change it.